include/sequence.h

```diff
@@ -161,6 +161,53 @@ typedef enum {
     /* 0x7 */ CHANNEL_IO_PORT_7
 } ChannelIOPort;
 
+typedef enum {
+    /* 0 */ VOL_SCALE_INDEX_BGM_MAIN,
+    /* 1 */ VOL_SCALE_INDEX_FANFARE,
+    /* 2 */ VOL_SCALE_INDEX_SFX,
+    /* 3 */ VOL_SCALE_INDEX_BGM_SUB,
+    /* 4 */ VOL_SCALE_INDEX_MAX
+} VolumeScaleIndex; // May be worth using SequencePlayerId instead
```

Comment:
If you look at the usage, they're indices `ScaleIndex` to `gActiveSeqs[seqPlayerIndex].volScales[scaleIndex]`.

And if you look how `Audio_SetVolumeScale(` is actually used in code, it seems like the intent was to have a volume scale for each `seqPlayerIndex` mapped to each other `seqPlayerIndex`:

For example, `seqPlayerIndex = SEQ_PLAYER_BGM_MAIN` can have it's volume affected by any of the other 4 seqPlayers (indexed by `ScaleIndex`).

You kinda have to look at all the usages of `Audio_SetVolumeScale(` and it's surrounding code to see that pattern, it's hard to explain

Reply:
I took a closer look and yeah I guess the idea is each index is per-player as "this is how whatever is playing on vol_scale_ corresponding player affects this other player". It will probably be more apparent when more things are named

Now should this be `SequencePlayerId`, idk. For now I'd say no, there's no absolute reason to merge them (like, they can shift independently) and it's easier to understand/read if separated I think